script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -352,12 +368,13 @@ This vampire has hidden themselves very skillfully. I can't sense their presence
 [sha:c92e0e200c30ecab32746c2f003da315eaf8e1f9]{
 -- Page 497, Offset 6929.
 -- 今回は手がかりがない状態だから、仕方なく昼間から街を歩いて調べていたの。
-This time I had no lead whatsoever, so I had no choice but to walk around the city in the daytime to investigate.
+I didn't have any leads this time so I had no choice but to walk around the city at daytime to investigate.
 }
 [sha:6091424f265763e545ca40ba79947f66790ded9e]{
 -- Page 497, Offset 6930.
 -- そうしていたら、通りすがりの殺人鬼に襲われちゃって、まあたいへん」
-And then, I was attacked by a passing serial killer, which was quite troublesome!"
+And as I was... this murderer happened by and killed me, what a pickle!"
+//pickles. also 殺人鬼 really should just be murderer, no need to imply anything else
```

Comment:
The problem with 殺人鬼 = serial killer is that Arcueid starts referring to him with this as a nickname (especially apparent in Ciel route when she goes full yan-yan) so I'd rather it be consistent from the start so people know where it came from

Because I don't want to translate "My precious little serial killer" or such in the future

Reply:
I'm not really convinced that's a real problem but this change is minor so I won't challenge it further.